libraries/ui/src/Nav.tsx

```diff
@@ -14,94 +13,152 @@ export type NavProps = React.PropsWithChildren<{
+    </div>
+    {children}
+  </div>
+);
+
+const LoginButtons: React.FC<{ className?: string }> = ({ className }) => (
+  <div className={clsx('nav__cta-container flex items-center', className)}>
+    <CTALinkOrButton className="nav__primary-cta" url="https://donate.stripe.com/5kA3fpgjpdJv6o89AA" isExternalUrl>Support us</CTALinkOrButton>
+    {/* <a className="nav__secondary-cta" href="https://course.aisafetyfundamentals.com/alignment?show=login" {...EXTERNAL_LINK_PROPS}>Log in</a>
+    <CTALinkOrButton className="nav__primary-cta" url="https://aisafetyfundamentals.com/" {...EXTERNAL_LINK_PROPS}>Get started for free</CTALinkOrButton> */}
+  </div>
+);
 
 export const Nav: React.FC<NavProps> = ({
   children, className, logo, courses,
 }) => {
-  const [isExpanded, setIsExpanded] = useState(false);
+  // Use a string rather than two booleans because `exploreExpanded = true`
+  // with `navExpanded = false` is not a valid state
+  const [expandedSections, setExpandedSections] = useState<'none' | 'nav' | 'nav-and-explore'>('none');
+  const navExpanded = expandedSections === 'nav' || expandedSections === 'nav-and-explore';
+  const exploreExpanded = expandedSections === 'nav-and-explore';
+
+  const onToggleExplore = useCallback(() => {
+    setExpandedSections((prev) => (prev === 'nav-and-explore' ? 'nav' : 'nav-and-explore'));
+  }, []);
+
+  const onToggleNav = useCallback(() => {
+    setExpandedSections((prev) => (prev === 'none' ? 'nav' : 'none'));
+  }, []);
 
   return (
-    <nav className={clsx(
-      'nav fixed z-50 w-full bg-color-canvas container-elevated',
-      'transition-all duration-300 left-1/2 -translate-x-1/2',
-      className,
-    )}
-    >
-      <div className="nav__container flex items-center justify-center w-full max-w-max-width h-20 mx-auto px-8">
-        <a href="/" className="nav__logo-link shrink-0 w-[200px]">
-          {logo ? <img className="nav__logo h-6 mr-auto" src={logo} alt="BlueDot Impact Logo" /> : <p className="nav_logo--placeholder h-8 mr-auto text-xl">BlueDot Impact</p>}
-        </a>
-        <div className="nav__links-container flex flex-grow justify-center items-center gap-[36px] relative">
-          <button
-            type="button"
-            onClick={() => setIsExpanded(!isExpanded)}
-            aria-expanded={isExpanded}
-            className="nav__dropdown-button flex items-center gap-2 hover:text-bluedot-normal"
+    <nav className={clsx('nav fixed z-50 w-full bg-cream-normal container-elevated', className)}>
+      <div className="nav__container w-full max-w-max-width mx-auto px-3 sm:px-6 lg:px-9">
+        <div className="nav__bar flex flex-grow justify-between items-center pl-3 h-[72px] sm:h-[100px]">
+          <a href="/" className="nav__logo-link shrink-0 w-[200px]">
+            {logo ? <img className="nav__logo h-6 mr-auto" src={logo} alt="BlueDot Impact Logo" /> : <p className="nav_logo--placeholder h-8 mr-auto text-xl">BlueDot Impact</p>}
```

Comment:
Should this now be `text-size-xl` re [this update](https://oliviamcgoffi-iwn5117.slack.com/archives/C089CN67W9L/p1738863847038509)?

Reply:
Yep good catch 🙏 . I ended up going with the closest elements that don't need an explicit `text-size-[]`:
![Screenshot 2025-02-08 at 12 31 22](https://github.com/user-attachments/assets/a58490b3-7ece-4218-99c7-0a38da48f3c6)